compiler/rustc_expand/src/base.rs

```diff
@@ -1136,17 +1136,17 @@ pub fn expr_to_string(
 }
 
 /// Non-fatally assert that `tts` is empty. Note that this function
```

Comment:
Rustdoc includes everything up until the first newline in the summary.
```suggestion
/// Non-fatally assert that `tts` is empty.
///
/// Note that this function
```